source/assets/sass/components/_alerts.scss

```
@@ -37,24 +34,28 @@
   }
 
   a {
-    border-bottom-color: rgba(theme.$sl-color--white, 0.5);
-    color: theme.$sl-color--white;
+    border-bottom-color: rgba(
+      brand.$sl-color--white,
+      var(--opacity-border, 0.5)
+    );
+    color: var(--sl-color--white);
     font-weight: typography.$sl-font-weight--bold;
 
     &:hover,
     &:focus {
       background: rgba(black, 0.0625);
```

I'm not sure on this, but based on the previous change, we maybe want:

```suggestion
      --color-link-bg: rgba(black, 0.0625);
```

---

fixed in https://github.com/oddbird/sass-site/pull/10/commits/29c9918fb5e406a4bbcc38f03a0739b84429f6a6.